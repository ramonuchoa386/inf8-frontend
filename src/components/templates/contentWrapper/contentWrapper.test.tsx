import React from 'react';
import { rendererCreate } from '../../../utils/test/test-utils';
import ContentWrapper from '.';

describe('Componente ContentWrapper', () => {
  test('Snapshot', () => {
    const template = rendererCreate(
      <ContentWrapper pageTitle='Title'>
        <div>Template test</div>
      </ContentWrapper>
    ).toJSON();

    expect(template).toMatchSnapshot();
  });
});
