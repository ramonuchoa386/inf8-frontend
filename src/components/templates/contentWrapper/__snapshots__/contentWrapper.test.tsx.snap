// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente ContentWrapper Snapshot 1`] = `
.c4 {
  border: none;
  cursor: pointer;
  color: #191918;
  background-color: transparent;
  text-transform: uppercase;
  font-weight: bold;
  padding: 8px;
  border-radius: calc(100 * 4px);
  font-size: 10px;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 4px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c4:hover {
  -webkit-transition: all 0.7s ease;
  transition: all 0.7s ease;
  color: #FFF;
  background-color: #191918;
}

.c4:hover svg {
  color: #FFF;
}

.c4:disabled {
  cursor: not-allowed;
  color: #CFCFC4;
  border-color: #CFCFC4;
}

.c4:disabled:hover {
  background-color: transparent;
}

.c13 {
  border: none;
  cursor: pointer;
  color: #FF805E;
  background-color: transparent;
  text-transform: uppercase;
  font-weight: bold;
  padding: 8px;
  border-radius: calc(100 * 4px);
  font-size: 10px;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 4px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c13:hover {
  -webkit-transition: all 0.7s ease;
  transition: all 0.7s ease;
  color: #FFF;
  background-color: #FF805E;
}

.c13:hover svg {
  color: #FFF;
}

.c13:disabled {
  cursor: not-allowed;
  color: #CFCFC4;
  border-color: #CFCFC4;
}

.c13:disabled:hover {
  background-color: transparent;
}

.c5 {
  border-radius: 4px;
  box-shadow: 0 1px 2px rgb(60 64 67 / 30%),0 1px 3px rgb(60 64 67 / 15%);
  padding: 16px;
}

.c3 {
  font-size: 0.875rem;
  line-height: 1.125rem;
  font-family: Inter;
  font-weight: 900;
  color: #464643;
}

.c10 {
  font-size: 1rem;
  line-height: 1.25rem;
  font-family: Inter;
  font-weight: 400;
  color: #464643;
}

.c11 {
  font-size: 1rem;
  line-height: 1.25rem;
  font-family: Inter;
  font-weight: 900;
  color: #464643;
}

.c8 {
  padding: 4px;
  border-radius: calc(100 * 4px);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  max-width: -webkit-fit-content;
  max-width: -moz-fit-content;
  max-width: fit-content;
  border: 1px solid #191918;
}

.c1 {
  width: 100%;
  background-color: #CFCFC4;
  padding: 16px 20px;
  -webkit-transition: all 0.4s ease-in-out;
  transition: all 0.4s ease-in-out;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  position: relative;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  gap: 8px;
}

.c6 {
  padding: 0px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  position: absolute;
  -webkit-transition: all 1s ease;
  transition: all 1s ease;
  z-index: 2;
  top: 0;
  visibility: hidden;
  opacity: 0;
  background-color: #FFF;
  border: 1px solid #B996FF;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 8px;
  padding: 16px;
  width: 100%;
}

.c7:first-of-type {
  border-bottom: 1px solid #CFCFC4;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
}

.c12 {
  font-size: 12px;
}

.c0 {
  width: 100%;
  height: 100vh;
  overflow: hidden;
}

.c14 {
  padding: 48px 20px 64px;
}

@media (min-width:768px) {
  .c1 {
    padding-left: 64px;
    padding-right: 64px;
  }
}

@media (min-width:1024px) {
  .c1 {
    padding-left: 48px;
    padding-right: 80px;
  }
}

@media (min-width:768px) {
  .c14 {
    padding-left: 64px;
    padding-right: 64px;
  }
}

@media (min-width:1366px) {
  .c14 {
    padding-left: 75px;
    padding-right: 75px;
    overflow: scroll;
    height: 100%;
  }

  .c14::-webkit-scrollbar {
    width: 8px;
  }

  .c14::-webkit-scrollbar-track {
    background: 'transparent';
  }

  .c14::-webkit-scrollbar-thumb {
    border-radius: calc(100 * 4px);
    background: #464643;
  }

  .c14::-webkit-scrollbar-thumb:hover {
    background: #191918;
  }
}

<div
  className="c0"
>
  <div />
  <header
    className="c1"
  >
    <section
      className="c2"
    >
      <svg
        className=""
        color="#191918"
        fill="currentColor"
        height="12px"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": "#191918",
          }
        }
        viewBox="0 0 24 24"
        width="12px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M12 2A10.13 10.13 0 0 0 2 12a10 10 0 0 0 4 7.92V20h.1a9.7 9.7 0 0 0 11.8 0h.1v-.08A10 10 0 0 0 22 12 10.13 10.13 0 0 0 12 2zM8.07 18.93A3 3 0 0 1 11 16.57h2a3 3 0 0 1 2.93 2.36 7.75 7.75 0 0 1-7.86 0zm9.54-1.29A5 5 0 0 0 13 14.57h-2a5 5 0 0 0-4.61 3.07A8 8 0 0 1 4 12a8.1 8.1 0 0 1 8-8 8.1 8.1 0 0 1 8 8 8 8 0 0 1-2.39 5.64z"
        />
        <path
          d="M12 6a3.91 3.91 0 0 0-4 4 3.91 3.91 0 0 0 4 4 3.91 3.91 0 0 0 4-4 3.91 3.91 0 0 0-4-4zm0 6a1.91 1.91 0 0 1-2-2 1.91 1.91 0 0 1 2-2 1.91 1.91 0 0 1 2 2 1.91 1.91 0 0 1-2 2z"
        />
      </svg>
      <p
        className="c3"
        size="small"
      >
        Portal do Administrador
      </p>
      <button
        className="c4 "
        onClick={[Function]}
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16.293 9.293 12 13.586 7.707 9.293l-1.414 1.414L12 16.414l5.707-5.707z"
          />
        </svg>
      </button>
    </section>
    <div
      className="c5 c6"
    >
      <div
        className="c7"
      >
        <div
          className="c8 "
        >
          <svg
            className=""
            color="#191918"
            fill="currentColor"
            height="12px"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "#191918",
              }
            }
            viewBox="0 0 24 24"
            width="12px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M12 2a5 5 0 1 0 5 5 5 5 0 0 0-5-5zm0 8a3 3 0 1 1 3-3 3 3 0 0 1-3 3zm9 11v-1a7 7 0 0 0-7-7h-4a7 7 0 0 0-7 7v1h2v-1a5 5 0 0 1 5-5h4a5 5 0 0 1 5 5v1z"
            />
          </svg>
        </div>
        <div
          className="c9"
        >
          <p
            className="c10"
          >
            Ol√°
          </p>
          <p
            className="c11 c12"
          >
            perfil:
          </p>
        </div>
      </div>
      <div
        className="c7"
        style={
          Object {
            "justifyContent": "space-between",
          }
        }
      >
        <button
          className="c13 "
          onClick={[Function]}
        >
          Sair
        </button>
      </div>
    </div>
  </header>
  <div
    className="c14"
  >
    <div>
      Template test
    </div>
  </div>
</div>
`;
